.github/workflows/docker-packaging.yml

```
@@ -0,0 +1,52 @@
+name: Centreon web build docker
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+    paths:
+      - ".github/workflows/docker-packaging.yml"
+      - ".github/docker/Dockerfile.packaging-centos7"
+      - ".github/docker/Dockerfile.packaging-alma8"
+      - ".github/docker/Dockerfile.packaging-bullseye"
+  pull_request:
+    paths:
+      - ".github/workflows/docker-packaging.yml"
+      - ".github/docker/Dockerfile.packaging-centos7"
+      - ".github/docker/Dockerfile.packaging-alma8"
+      - ".github/docker/Dockerfile.packaging-bullseye"
+
+jobs:
+  get-version:
+    uses: ./.github/workflows/get-version.yml
+    with:
+      version_file: centreon/www/install/insertBaseConf.sql
+
+  create-and-push-docker:
```

```suggestion
  dockerize:
```

in order to remain consistent